backend_kivy.py

```diff
@@ -1244,7 +1259,8 @@ def set_window_title(self, title):
         EventLoop.window.title = title
 
     def resize(self, w, h):
-        Window.size(w, h)
+        if (w > 0) and (h > 0):
+            EventLoop.window.size = (w, h)
```

Comment:
`Window.size = w, h` is probably better (although your current way works just as fine).